package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "lambda_mail_parser",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "async": "^2.6.1",
+    "aws-sdk": "^2.385.0",
+    "gm": "^1.23.1",
+    "mailparser": "^2.4.3",
+    "mailparser-mit": "^0.6.2",
```

Comment:
Why are we adding the MIT fork? We want to use the main library, not a fork.

Reply:
Doing the EUPL only because we’re not using the MIT fork.

Reply:
Forgot to remove that, I was experimenting